### ISSUE TITLE
Fix flickering ellipsis (...) button on tab groups

### ISSUE THREAD

**Issue by tjcouch-sil** [paranext/paranext-core#66]
Sometimes, when you resize tab groups just the right amount, the ... button that shows up when there are too many tabs to display flickers on and off the screen. This is probably some small bug in `rc-dock` that has to do with not testing the widths correctly according to how we're doing things. We should use the patch package on rc-dock.

**Comment by FoolRunning** [comment]
I'm fairly certain I've found the place in the rc-tabs code that is causing the problem. See **TabNavList/index.tsx** and the ``updateTabSizes`` method.
However, I can't consistently reproduce the problem and I can't tell if the patching system is even working so I'm giving up and letting someone else take a stab at fixing this.

**Comment by irahopkinson** [comment]
@FoolRunning if you have a branch with any changes you tried (even if they aren't working) I'd appreciate you pushing it.

**Comment by FoolRunning** [comment]
@irahopkinson, Done. Pushed to to the ``66_flickeringButton`` branch. There isn't much there as most of the time was trying to reproduce it and finding the code inside of rc_tabs that might be causing the problem.

**Comment by irahopkinson** [comment]
@FoolRunning err... did you push the branch - I'm not seeing it yet?

**Comment by irahopkinson** [comment]
Even with the fixes above there are scenarios where this issue still occurs, see this [CodeSandbox](https://codesandbox.io/p/sandbox/rc-dock-ellipsis-button-flicker-fixed-5j7kkl?file=%2Fsrc%2FRoot.css%3A25%2C50) where there are multiple tabs in 2 side-by-side panels (I created this and the [base issue](https://codesandbox.io/p/sandbox/rc-dock-ellipsis-button-flicker-tgth6n?file=%2Fsrc%2FRoot.css%3A25%2C49) as I was going to report it to `rc-dock`).

**Comment by irahopkinson** [comment]
After more investigation, I can't actually reproduce the issue in Platform.Bible using the scenario described in my previous comment even though I could in the CodeSandbox. This issue may still cause us an issue in the future but I don't think it's worth spending more time on now.

In the meantime, I have reported [the issue](https://github.com/ticlo/rc-dock/issues/215) to `rc-dock`.
Closing again.
